docs/Splashkit/Applications/Build a COOL game/Research & Findings/Sound Effects/Sound Effects.md

```diff
@@ -0,0 +1,20 @@
+# Sound Effects
+## Introduction
+When building COOL game, sound effect is one of the core element. For our different types of arcade games, we found the following royalty-free sound effects that can be used.
+
+There are game over music (and potentially loop-able game music), jumping noise (for jump), collision/hit noise, dice roll sound, and some audio saying "start!" and "game over", kick/punch sound, block moving noise.
+
+## Game1 - Jump
+All sound effects for Jump
+[here]()
```

Comment:
Should this be linked to the folders?
If so, you will need to make an index.md in each folder. This could be a table of contents linking to each sound